### ISSUE TITLE
grep -A and SC2126

### ISSUE THREAD

**Issue by matteocorti** [koalaman/shellcheck#2585]
#### For bugs
- Rule Id: SC2126
- My shellcheck version: 0.8.0
- [X] The rule's wiki page does not already cover this (e.g. https://shellcheck.net/wiki/SC2086)
- [X] I tried on https://www.shellcheck.net/ and verified that this is still a problem on the latest commit

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/bin/sh

grep -A 1000 -c pattern file | wc -l
```

#### Here's what shellcheck currently says:

```
Consider using grep -c instead of grep|wc -l.
```

#### Here's what I wanted or expected to see:

In this case, ```wc``` is not counting the matches lines but also the (max 1000) lines following the match. The same applies for ```-B``` and ```-C```.

The warning should be suppressed if ```grep``` is called with ```-A```, ```-B``` or ```-C```.

**Comment by tr4nshum4n** [comment]
Hello!

I entered your code at [https://www.shellcheck.net/](https://www.shellcheck.net/), and did not see the message you indicated.

It looks like this was addressed in [PR 2414](https://github.com/koalaman/shellcheck/pull/2414).

Did you run through shellcheck recently?

**Comment by matteocorti** [comment]
Thanks, you are right. I am now installing from source (as the PR is not yet released).
